go.mod

```diff
@@ -175,9 +175,10 @@ require (
 	github.com/golang-jwt/jwt/v4 v4.5.0 // indirect
 	github.com/google/btree v1.0.1 // indirect
 	github.com/google/pprof v0.0.0-20210720184732-4bb14d4b1be1 // indirect
+	github.com/google/s2a-go v0.1.5 // indirect
```

Comment:
this adds a lot of new code, and a new transitive dependency to google.golang.org/appengine (and actually pulls in appengine code) which we don't want

it looks like that gets pulled in by this:
```go
vendor/google.golang.org/api/internal/cba.go
...
"github.com/google/s2a-go"
```

I'm not confident this will drop out once https://github.com/kubernetes/kubernetes/issues/113366 is resolved.

Reply:
add "github.com/google/s2a-go" and "google.golang.org/appengine" to the unwanted dependencies list so we can understand what is referencing them specifically, and whether the genproto cleanup and GCE cloud provider removal will let these drop out

Reply:
commented where this got introduced in https://github.com/googleapis/google-api-go-client/pull/1874/files#r1312969466, will ask more questions there

Reply:
I opened https://github.com/kubernetes/kubernetes/pull/120353 to make existing links to these visible. Once that merges, you can rebase on that and we'll see what this is doing to our transitive references

Reply:
added "github.com/google/s2a-go" and "google.golang.org/appengine" to unwanted